### DISCUSSION TITLE
Lesson 11 - testConvertSvgToUri reverts

### DISCUSSION THREAD

**Discussion by RichardPalladino** [Cyfrin/foundry-full-course-cu#463] [Q&A] [answered]
I'm working through the course and Patrick has us test the deployment script with `testConvertSvgToUri()`.

I followed all the steps but my expected URI is:
`data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCI+Cjx0ZXh0IHg9IjAiIHk9IjE1IiBmaWxsPSJibGFjayI+SGkhIFlvdXIgYnJvd3NlciBkZWNvZGVkIHRoaXM8L3RleHQ+Cjwvc3ZnPg==`

The actual URI is:
`data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCI+PHRleHQgeD0iMCIgeT0iMTUiIGZpbGw9ImJsYWNrIj5IaSEgWW91ciBicm93c2VyIGRlY29kZWQgdGhpczwvdGV4dD48L3N2Zz4=`

My confusion is that BOTH of those URIs result in the same output.  They also have the same first set of characters.  Only the data following the plus (+) sign is different.

In addition, the test is not part of the lesson's code.  It has been removed from the test set, as far as I can tell.

Anyone have an idea of where I may be going wrong?  (I'm running Linux on WSL... but the base64 CLI utility should function consistently no matter the platform.)

**Comment by shikhar229169** [comment, marked as answer]
Consider referring to the following discussions, I have already answered it there:
https://github.com/Cyfrin/foundry-full-course-f23/discussions/248
https://github.com/Cyfrin/foundry-full-course-f23/discussions/266

**Reply by RichardPalladino** [reply]
Thanks.  I'm familiar enough with base64 so that makes sense.  I don't see any newline character in the file, but I blame Winblows OS.

I copied the SVG line into a new file that I made in WSL and it all worked out.
